libraries/tests/mbed/serial_nc_rx/main.cpp

```diff
@@ -2,29 +2,36 @@
 #include "test_env.h"
 
 int main() {
+    Serial *pc = new Serial(USBTX, USBRX);
```

Comment:
I wonder if it would be better to do hosttest macros first, flush the buffer, and then create a new serial?